ocs_ci/ocs/fiojob.py

```diff
@@ -545,9 +556,14 @@ def is_storage_reclaimed():
         label = f'fixture={fixture_name}'
         ocp_pv.add_label(pv_name, label)
     else:
-        # Without checksum, we just need to make sure that data were deleted
-        # and wait for this to happen to avoid conflicts with tests executed
+        # Without checksum, we just need to make sure that the data will be deleted.
+        # If 'delete_fio_data_after_test' is False and 'request' object is provided
+        # Wait until the end of the tests, and then delete the data.
+        if not delete_fio_data_after_test and request:
+            request.addfinalizer(lambda: delete_fio_data(fio_job_file, is_storage_reclaimed))
+        # Wait for the data to be deleted, to avoid conflicts with tests executed
```

Comment:
This change obstructs the original meaning a bit.

Reply:
ok, I will change it to what it was before. I really wasn't sure about that myself. I am just thinking about what we should do if the user didn't provide the 'request' param.